### PR TITLE
Increase test coverage for `int.jl`

### PR DESCRIPTION
**Pull request by vmpyr**

### CODE REVIEW

File: test/int.jl [hunk 1]
```diff
@@ -300,6 +300,29 @@ end
...
+    @test typemin(Int32  ) === Int32(-2147483648)
+    @test typemax(Int32  ) === Int32(2147483647)
+    @test typemin(UInt32 ) === UInt32(0)
+    @test typemax(UInt32 ) === UInt32(4294967295)
+    @test typemin(Int64  ) === Int64(-9223372036854775808)
+    @test typemax(Int64  ) === Int64(9223372036854775807)
+    @test typemin(UInt64 ) === UInt64(0)
+    @test typemax(UInt64 ) === UInt64(0xffffffffffffffff)
+    @test typemin(UInt128) === UInt128(0)
+    @test typemax(UInt128) === UInt128(0xffffffffffffffffffffffffffffffff)
```
Comment (inkydragon) [inline test/int.jl:321]:
```suggestion
    @test typemax(UInt64 ) === UInt64(0xffff_ffff_ffff_ffff)
    @test typemin(UInt128) === UInt128(0)
    @test typemax(UInt128) === UInt128(0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff) 
```

Maybe it's more readable that way.

**Comment by giordano** [comment]
@vmpyr to signal a PR is work-in-progress it's better to open it a [draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) and mark it as "ready for review" when you're done, rather than putting "WIP" and "Work Complete" in the title.

**Comment by vmpyr** [comment]
@giordano okay, I'll keep that in mind. Thanks!

**Comment by inkydragon** [comment]
NOTE: Those tests will cover L859~L878 https://coveralls.io/builds/56776531/source?filename=base%2Fint.jl#L859

**Pull request merged**